Add CL item for sprint assignment to promotion issue template (#6930) (#7098)
DataBiosphere/azul: .github/ISSUE_TEMPLATE/promotion.md

_period: 7 days
---
- [ ] The title of this issue matches `Promotion yyyy-mm-dd`

> - [ ] The sprint of this issue matches the date in the title

Reviewer: Consider …
```suggestion
- [ ] The sprint of this issue corresponds to the date in the title
```
… and moving it to the bottom. We may not have a sprint until after the promotion is decided.

- [ ] For `anvilprod`
  - [ ] System administrator and operator determined the commit to be promoted
  - [ ] Operator created the promotion PR

> - [ ] The sprint of this issue corresponds to the date in the title

Reviewer 2: ```suggestion
- [ ] This issue is part of the sprint that includes the date from its title
```

> - [ ] This issue is part of the sprint that includes the date from its title

Reviewer 2: ```suggestion
- [ ] This issue is part of the sprint that includes the date from the issue title
```
